### ISSUE TITLE
Concerns around syntax

### ISSUE THREAD

**Issue by annevk** [WICG/feature-policy#100]
A couple folks at Mozilla raised the concern that the syntax here is rather complex. We're effectively inventing yet another parser/model/serializer.

Assuming this complexity is justified by the complexity needed for the model, is there a way that we could perhaps reuse an existing parser that would give us a similar model?

(For reference, @domenic and I ended up working on around 8k tests (inc generated) just for MIME types. There's quite a bit of complexity in even smallish constructs.)

**Comment by domenic** [comment]
I thought the idea was to reuse CSP syntax, to address exactly this concern.

**Comment by clelland** [comment]
Yes, we switched from the JSON syntax for exactly this reason -- Chrome security folks were concerned about introducing another parser for the allow attribute, and we were able to map it to a CSP-style syntax, so we switched to that.

**Comment by annevk** [comment]
Okay, so let's use this issue to make sure they actually use a single shared parser, also in the specification? We could potentially uplift it to the Infra Standard, if neither specification wants to depend on the other.
